dts/bindings/interrupt-controller/nxp,irqsteer-intc.yaml

```diff
@@ -0,0 +1,16 @@
+description: i.MX DSP interrupt controller
+
+compatible: "nxp,irqsteer-intc"
+
+include: [interrupt-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
```

Comment:
this is required?

Reply:
Now, I realized is not :)
I removed it!